### PR TITLE
feat: pass jwt token for getting bookmarkId data

### PR DESCRIPTION
**Pull request by Aylie-Chou**
# Issue
[[會員制] 我的閱讀 - 最新頁新增收藏功能](https://app.asana.com/0/1203699264568808/1206201445527861/f)

# Notice
- toggle bookmark on UI would be developed after common design system component [PR](https://github.com/twreporter/twreporter-npm-packages/pull/485) merged

# Dependency
https://github.com/twreporter/go-api/pull/657
https://github.com/twreporter/twreporter-npm-packages/pull/486
https://github.com/twreporter/twreporter-npm-packages/pull/485
https://github.com/twreporter/twreporter-npm-packages/pull/488

### CODE REVIEW

**Review by liruchen32 — COMMENTED** [review]
Do we need to show `snackbar` after toggle bookmark ?

File: src/containers/Latest.js [hunk 1]
```diff
@@ -236,6 +259,7 @@ const Latest = ({
           <CardList
             data={posts}
             showSpinner={true}
+            showIsBookmarked={true}
```
[the remark this answers is not in the document]
Comment (Aylie-Chou) [reply src/containers/Latest.js]: This is still under discussion, but I hide it when user is not authed for now ([0678a07](https://github.com/twreporter/twreporter-react/pull/2724/commits/0678a0767fcd851ea0da5f85d69608786deef327))

**Comment by Aylie-Chou** [comment]
updated ([0678a07](https://github.com/twreporter/twreporter-react/pull/2724/commits/0678a0767fcd851ea0da5f85d69608786deef327))

- add general snackbar & pass `toastr` in CoreContext
- hide bookmark in latest page if user is not authed

**Review by liruchen32 — APPROVED** [review]
LGTM

**Pull request merged**